src/Matrix.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) 2019 Shapelets.io
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System;
+using Khiva.Interop;
+
+namespace Khiva
+{
+    /// <summary>
+    /// Khiva Matrix Profile class containing matrix profile methods.
+    /// </summary>
+    public static class Matrix
+    {
+        /// <summary>
+        /// Primitive of the findBestNDiscords function.
+        /// </summary>
+        /// <param name="profile">The matrix profile containing the minimum distance of each subsequence</param>
```

Comment:
missing points in all these lines:

        /// <param name="profile">The matrix profile containing the minimum distance of each subsequence</param>
        /// <param name="index">The matrix profile index containing the index of the most similar subsequence</param>
        /// <param name="m">Length of the matrix profile</param>
        /// <param name="n">Number of discords to extract</param>